st2actions/st2actions/workflows/workflows.py

```diff
@@ -60,11 +64,15 @@ def process(self, wf_ex_db):
 
         # If there is no new tasks, update execution records to handle possible completion.
         if not next_tasks:
+            LOG.info('[%s] No next tasks identified for workflow execution.', wf_ac_ex_id)
```

Comment:
While testing the changes I noticed we could also include actual objects using `extra` logger attribute.

Not a blocker for now, but it means log messages would include more context and perhaps make debugging / troubleshooting easier in some scenarios.

Reply:
No.... I don't like including actual objects in log messages. It makes the log way too noisy. I prefer a pointer that the operator can selectively query from st2.